script/dolphinscheduler-daemon.sh

```diff
@@ -113,19 +108,19 @@ case $startStop in
       ;;
 
   (status)
-    # more details about the status can be added later
-    serverCount=`ps -ef | grep "$DOLPHINSCHEDULER_HOME" | grep "$CLASS" | grep -v "grep" | wc -l`
-    state="STOP"
-    #  font color - red
-    state="[ \033[1;31m $state \033[0m ]"
-    if [[ $serverCount -gt 0 ]];then
-      state="RUNNING"
-      # font color - green
-      state="[ \033[1;32m $state \033[0m ]"
-    fi
-    echo -e "$command  $state"
-    ;;
-
+      state="STOP"
+      state="[ \033[1;31m $state \033[0m ]"
+      if [ -f $pid ]; then
+        TARGET_PID=`cat $pid`
+        serverCount=`ps -p $TARGET_PID | wc -l`
+        if [[ $serverCount -gt 1 ]];then
+          state="RUNNING"
+          state="[ \033[1;32m $state \033[0m ]"
+        fi
+      echo -e "$command  $state"
+      fi
+      ;;
+      
```

Comment:
I think we should not do this. The previous version is enough. Please rollback it.

Reply:
This is the,This is verified by obtaining the PID of each service from the PID file.

Reply:
Just as you said

Reply:
Yes. But the old method also realizes this function. We should try to reduce large-scale modifications to ensure stability.

Reply:
@SbloodyS I think this is better than the old, more beautiful ^_^

Reply:
This change is want to used the pid as filter rather than use the process name.
If the pid file has been deleted, this method may return an incorrect status.